.travis.yml

```diff
@@ -4,6 +4,8 @@ xcode_scheme: Tests
 osx_image: xcode9.1
 
 before_install:
+- gem uninstall --force -x -i /Users/travis/.rvm/gems/ruby-2.4.2@global xcodeproj
+- gem install -q -N -f --no-update-sources xcodeproj:1.5.7
 - gem update fastlane --no-ri --no-rdoc --no-document
```

Comment:
For a dirty workaround like this please add a FIXME comment that references the responsible ticket https://github.com/CocoaPods/Xcodeproj/issues/572 so that we remember to remove the workaround.